libraries/models.py

```diff
@@ -48,6 +49,12 @@ class CommitAuthor(models.Model):
     avatar_url = models.URLField(null=True, max_length=100)
     github_profile_url = models.URLField(null=True, max_length=100)
 
+    @property
+    def display_name(self):
+        User = get_user_model()
+        u = User.get_user_by_github_url(self.github_profile_url)
+        return getattr(u, "display_name", None)
```

Comment:
Wouldn't this be simpler? Is there an actual use case for this property returning `None`?

```suggestion
        return u.display_name or self.name
```

Reply:
get_user_by_github_url doesn't necessarily always return a user - someone might contribute on libraries and not have an account on the site - so returns None, which means u.display_name wouldn't exist exist. 

It could return self.name instead of none though so I'll change it that way. I was thinking it might be better to keep them separate but I guess it won't matter too much in use.